content/attacks/posts/2021-05-19-PancakeBunny.md

```diff
@@ -0,0 +1,57 @@
+---
+date: 2021-05-19
+target-entities: PancakeBunny
+entity-types: DeFi
+attack-types: Flash Loan Attack
+title: "PancakeBunny suffers a flash loan attack"
+---
+
+## Summary
+
+On May 19, 2021 PancakeBunny, a yield farming aggregator built on Binance Smart Chain decentralized finance (DeFi) protocol, suffered a flash loan attack. According to [the SlowMist report they shared after investigating the incident](https://slowmist.medium.com/slowmist-pancakebunny-hack-analysis-4a708e284693) 
+> The attacker first borrows a huge amount of WBNB tokens from the multiple liquidity pools of PancakeSwap and borrows a huge amount of WBNB tokens from the Fortube project.
+>
+These assets were used to manipulate the price of BUNNY/BNB and USDT/BNB.
+
+Cofounder of Halborn Rob Behnke describes the main steps of the manipulating as follows: minting LP tokens as a result of depositing a pair of tokens; swapping a large amount of one token of this pair for another to modify an exchange rate; finally, exchanging LP tokens for share of the invested pair, exploiting modified prices. This double price manipulation allowed the hacker to get profit 
+> when extracting value or exchanging BUNNY for BNB the attacker received more tokens than they "should" have. As a result, after paying off the flash loan, the attacker had 114,631 WBNB left over.
+>
+> -- Rob Behnke
+[Source](https://www.halborn.com/blog/post/explained-the-pancakebunny-protocol-hack-may-2021)
+>
+Another part of the hacker's profit was the remainder of BUNNY tokens he acquired according to the BunnyMinterV2 contract.
+
+## Attackers
+
+The attacker remain unidentified. 
+Attacker's wallet address:
+[0xa0ACC61547f6bd066f7c9663C17A312b6Ad7E187](https://bscscan.com/address/0xa0acc61547f6bd066f7c9663c17a312b6ad7e187)
+
+## Losses
+
+The amount of stolen assets [was estimated at about $200 million at prices at the time. BUNNY token lost around 96% of its value.](https://cointelegraph.com/news/pancakebunny-tanks-96-following-200m-flash-loan-exploit)
+
+## Timeline
+
+The timeline of the incident with more details is provided on [the PancakeBunny Medium blog](https://pancakebunny.medium.com/hello-bunny-fam-a7bf0c7a07ba)
+
+**May 19, 2021**
+- **10:31 PM UTC:** 1BNB deposited on the USDT/BNB Flip Vault to stage the attack
```

Comment:
Delete the line

Reply:
I see now, you mean this one "1BNB deposited on the USDT/BNB Flip Vault to stage the attack"?
Most likely i misunderstood first and moved the source's link below